tutorials/beginner/logs.md

```diff
@@ -6,11 +6,17 @@ layout: tutorial-beginner
 
 # Looking to the logs
 
-While using openHAB, we will need to look the the logs generated by the software.
+While using openHAB, we will need to look at the the logs generated by the software to see if there are any issues that need your attention.
 
-Two log files are going to interest us.
+Two log files are going to interest us. The two files are called openhab.log and events.log with the events.log file showing what is happening on the Openhab event bus like sensors changing states, and the openhab.log file which shows the output of all bindings and addons.
```

Comment:
Maybe we should rewrite this and remove the exiting sentence.

An idea to start with: (What do you think about it?)

---

There are two files going to interest us, which will be `openhab.log` and `events.log`.
`openhab.log` shows the output of all bindings and addons.
`events.log` displays whats happening on the openHAB event bus, like changing states of sensors.

---

Please also keep in mind, that we are writing only one sentence per line.
This is needed/useful to keep a better track of changes in GitHub.